src/main/java/ro/societateahermes/backendservice/controller/FormController.java

```diff
@@ -0,0 +1,27 @@
+package ro.societateahermes.backendservice.controller;
+
+import org.springframework.web.bind.annotation.*;
+import ro.societateahermes.backendservice.entities.form.Form;
+import ro.societateahermes.backendservice.service.serviceImplementation.FormService;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/form")
+public class FormController {
+    private final FormService formService;
+
+    public FormController(FormService formService) {
```

Comment:
Foloseste @Autowired la constructor sau la field. Asa o instanta a lucrului la care dai autowired va fi injectata automat in clasa